peps/pep-0772.rst

```diff
@@ -0,0 +1,1612 @@
+PEP: 772
+Title: A Python Installation Manager for Windows
+Author: Steve Dower
+PEP-Delegate: TBD
+Discussions-To: https://discuss.python.org/t/pep-772-a-python-installation-manager-for-windows/77900
+Status: Draft
+Type: Standards Track
+Topic: Release
+Created: 21-Jan-2025
+Post-History: 18-Dec-2024
```

Comment:
We can link to Post-History now, but I don't have the link for your old topic -- please edit the suggestion to add it!
```suggestion
Author: Steve Dower
Discussions-To: https://discuss.python.org/t/77900/
Status: Draft
Type: Standards Track
Topic: Release
Created: 21-Jan-2025
Post-History:
   `18-Dec-2024 <https://discuss.python.org/t/77900/>`__
   `21-Jan-2025 <https://discuss.python.org/t/77900/>`__
```